src/webgpu/shader/execution/expression/call/builtin/subgroupBallot.spec.ts

```diff
@@ -344,4 +348,315 @@ fn main(@builtin(subgroup_size) subgroupSize : u32,
     await runTest(t, wgsl, testcase.filter, testcase.expect, false);
   });
 
-g.test('fragment').unimplemented();
+// Filters should always skip the last row and column.
```

Comment:
nit: recommend saying why:  To avoid potential interference from helper invocations.

Reply:
and now I see it's commented on below too.  Nice.